DevPod.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>sh.loft.devpod</id>
+
+  <name>Devpod</name>
+  <summary>Codespaces but open-source, client-only and unopinionated: Works with any IDE and lets you use any cloud, kubernetes or just localhost docker.</summary>
```

Comment:
summary is not description, try to keep it within 30-35 words and add the rest to description.